### ISSUE TITLE
Provide unified profile-level Records

### ISSUE THREAD

**Issue by joshhunt** [Bungie-net/api#758]
While the data for Records exist at both the Profile and Character level, I believe users expect Triumphs to operate profile-wide, so building out something that operates similarly to the in-game Triumphs viewer is trickier than it needs to be IMHO because of how you need to merge in character records with profile records. I'm always anxious that when I do this I'm merging incorrectly, or a character with that triumph not completed overrides a character that has completed it.

Could the ProfileRecords summarise the character-level records, similar to how its done in game?

**Comment by vthornheart-bng** [comment]
Indeed, this is something I struggled with philosophically while making the system.  The fears I have is that there's actually no hard-set restriction keeping Triumphs from being character-specific: and indeed, some portions of the Triumphs state even now is character-specific (such as visibility IIRC).

Many times, when I assume that something's going to be kept at a certain scope in the game and there's not a hard-coded restriction keeping someone from changing it, it gets changed and causes us to have to pivot in unexpected ways.  It was that paranoia that made me make the Triumphs as they are currently... but I'll give it some thought, I may have been overly paranoid.
